jenkins/data-prepper/Jenkinsfile

```diff
@@ -0,0 +1,80 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-v1'
```

Comment:
We also need to find the correct image. We need a Docker image with JDK 14 for Data Prepper, and Python/Pip for the signArtifacts task.